stable/pipelines/templates/ml-pipeline/ui/deployment.yaml

```diff
@@ -17,6 +17,8 @@ spec:
       labels:
         component: ml-pipeline-ui
 {{ include "pipelines.commonLabels" . | indent 8 }}
+      annotations:
+        cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
```

Comment:
do we need that annotation?

Reply:
It enables scale-down, so I removed it from all deployment yamls.